diff --git a/fluidsynth/src/midi/fluid_midi.h b/fluidsynth/src/midi/fluid_midi.h
index 90fcef7..f84a883 100644
--- a/fluidsynth/src/midi/fluid_midi.h
+++ b/fluidsynth/src/midi/fluid_midi.h
@@ -247,11 +247,11 @@ struct _fluid_track_t {
 
 typedef struct _fluid_track_t fluid_track_t;
 
-fluid_track_t* new_fluid_track(int num);
+FLUIDSYNTH_API fluid_track_t* new_fluid_track(int num);
 int delete_fluid_track(fluid_track_t* track);
 int fluid_track_set_name(fluid_track_t* track, char* name);
 char* fluid_track_get_name(fluid_track_t* track);
-int fluid_track_add_event(fluid_track_t* track, fluid_midi_event_t* evt);
+FLUIDSYNTH_API int fluid_track_add_event(fluid_track_t* track, fluid_midi_event_t* evt);
 fluid_midi_event_t* fluid_track_first_event(fluid_track_t* track);
 fluid_midi_event_t* fluid_track_next_event(fluid_track_t* track);
 int fluid_track_get_duration(fluid_track_t* track);
@@ -309,8 +309,8 @@ struct _fluid_player_t {
   void* playback_userdata; /* pointer to user-defined data passed to playback_callback function */
 };
 
-int fluid_player_add_track(fluid_player_t* player, fluid_track_t* track);
-int fluid_player_callback(void* data, unsigned int msec);
+FLUIDSYNTH_API int fluid_player_add_track(fluid_player_t* player, fluid_track_t* track);
+FLUIDSYNTH_API int fluid_player_callback(void* data, unsigned int msec);
 int fluid_player_count_tracks(fluid_player_t* player);
 fluid_track_t* fluid_player_get_track(fluid_player_t* player, int i);
 int fluid_player_reset(fluid_player_t* player);
diff --git a/fluidsynth/src/utils/fluid_sys.h b/fluidsynth/src/utils/fluid_sys.h
index 4953515..2b1eb3d 100644
--- a/fluidsynth/src/utils/fluid_sys.h
+++ b/fluidsynth/src/utils/fluid_sys.h
@@ -116,11 +116,11 @@ typedef int (*fluid_timer_callback_t)(void* data, unsigned int msec);
 
 typedef struct _fluid_timer_t fluid_timer_t;
 
-fluid_timer_t* new_fluid_timer(int msec, fluid_timer_callback_t callback,
+FLUIDSYNTH_API fluid_timer_t* new_fluid_timer(int msec, fluid_timer_callback_t callback,
                                void* data, int new_thread, int auto_destroy,
                                int high_priority);
 
-int delete_fluid_timer(fluid_timer_t* timer);
+FLUIDSYNTH_API int delete_fluid_timer(fluid_timer_t* timer);
 int fluid_timer_join(fluid_timer_t* timer);
 int fluid_timer_stop(fluid_timer_t* timer);
 
